tests/test_flatten.py

```diff
@@ -19,7 +28,11 @@ def teste_flatten_with_emptylist():
     assert flatten([]) == []
 
 
-# test the flatten function with a dictionary mocking it to assert _flatten_dict function is being called
+def teste_flatten_with_emptyDataFrame():
```

Comment:
Any reason to ignore PEP8 in the [naming](https://peps.python.org/pep-0008/#function-and-variable-names)?

```suggestiotn
def teste_flatten_with_empty_data_frame():
```

Reply:
Sorry. Mental issues here...
By the way, should I improve the name with the expected result?
like `teste_flatten_with_empty_data_frame_return_empty_data_frame` or `teste_flatten_with_empty_data_frame_yearly_return`?

Reply:
You can, but no need to. The test is short enough to make that clear : )